llpc/test/shaderdb/PipelineCs_RelocConst.pipe

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
Some comments describing what you are trying to test will be useful.

Reply:
Done.